go/test/endtoend/mysqlctld/mysqlctld_test.go

```diff
@@ -134,10 +135,29 @@ func initCluster(shardNames []string, totalTabletsRequired int) error {
 	return nil
 }
 
+const defaultOperationTimeout = 60 * time.Second
+const defeaultRetryDelay = 3 * time.Second
+
+func waitForMysqlctldShutdown(t *testing.T, tab *cluster.Vttablet) bool {
+	tmr := time.NewTimer(defaultOperationTimeout)
```

Comment:
It's a test, but we should still stop it in a defer:
```
defer tmr.Stop()
```

Reply:
fixed